src/main/java/teammates/ui/output/NotificationData.java

```diff
@@ -37,11 +37,11 @@ public String getNotificationId() {
         return this.notificationId;
     }
 
-    public Long getStartTimestamp() {
+    public long getStartTimestamp() {
```

Comment:
I made this change because I changed the input from the user is changed to `long` in `CreateNotificationRequest.java` as there was no reason to take a `Long`, I can refactor `createdAt` and `updatedAt` if I find there are no dependencies

Reply:
Please update for the created/updated too